sprocs/submissions_insert.sql

```diff
@@ -38,12 +39,16 @@ BEGIN
     IF NOT FOUND THEN RAISE EXCEPTION 'invalid variant_id = %', variant_id; END IF;
     
     -- we must have a variant, but we might not have an assessment_instance
-    SELECT              iq.id,                  ai.id
-    INTO instance_question_id, assessment_instance_id
+    SELECT iq.id,                ai.id,
+           iq.requires_manual_grading OR q.grading_method = 'Manual'
+    INTO   instance_question_id, assessment_instance_id,
+           new_requires_manual_grading
```

Comment:
Nit, as long as we're modifying this code:
```suggestion
    SELECT
        iq.id,
        ai.id,
        iq.requires_manual_grading OR q.grading_method = 'Manual'
    INTO
        instance_question_id,
        assessment_instance_id,
        new_requires_manual_grading
```